### ISSUE TITLE
Plugin name / permalink slug currently conflicts with Cloudflare Stream

### ISSUE THREAD

**Issue by B-Interactive** [B-Interactive/cloudflare-stream-wordpress#24]
In preparation for plugin submission to WordPress for review, [the current plugin name](https://github.com/B-Interactive/cloudflare-stream-wordpress/blob/7d6c665ba6ac69e595146f6191a000e741c6dbbc/cloudflare-stream.php#L3) (which determines its permalink slug) conflicts with [the legacy plugin](https://github.com/cloudflare/stream-wordpress/blob/e949271e91bb402d5befe73acd099fe73b1e0fae/cloudflare-stream.php#L3).

```
 * Plugin Name: Cloudflare Stream
```

Which upon submission would _attempt_ to resolve to the legacy plugin's permalink, and conflict:
`https://wordpress.org/plugins/cloudflare-stream/`

[WordPress - Plugin Developer FAQ - What will my plugin permalink (slug) be?](https://developer.wordpress.org/plugins/wordpress-org/plugin-developer-faq/#what-will-my-plugin-permalink-slug-be)

> This is populated based on the value of Plugin Name in your main plugin file...

>If you set yours as Plugin Name: Boaty McBoatface then your URL will be wordpress.org/plugins/boaty-mcboatface and your slug will be boaty-mcboatface

>You can’t have two plugins with the same permalink so you need to pick a new one.

A new name is therefore required.  Requesting ideas!...

**Comment by davidmpurdy** [comment]
Yeah, the name is tricky. Also can't start with "Cloudflare Stream"...

If we think it will continue to focus on embedding (vs. Stream video post types, which is on my roadmap but may be a separate plugin), maybe something like "Video embedding for Cloudflare Stream"?

Or something generic like "Video integration for Cloudflare Stream" or "Video tools for Cloudflare Stream" (I don't quite have the right idea for that second word...)
